src/SprykerEco/Zed/Payone/Business/Payment/MethodSender/AbstractPayoneRequestSender.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Payone\Business\Payment\MethodSender;
+
+use Orm\Zed\Payone\Persistence\SpyPaymentPayone;
+use Orm\Zed\Payone\Persistence\SpyPaymentPayoneApiLog;
+use SprykerEco\Zed\Payone\Business\Api\Request\Container\AbstractRequestContainer;
+use SprykerEco\Zed\Payone\Business\Api\Request\Container\CaptureContainer;
+use SprykerEco\Zed\Payone\Business\Api\Request\Container\DebitContainer;
+use SprykerEco\Zed\Payone\Business\Api\Request\Container\RefundContainer;
+use SprykerEco\Zed\Payone\Business\Payment\PaymentMapperReaderInterface;
+use SprykerEco\Zed\Payone\Business\Payment\PaymentMethodMapperInterface;
+use SprykerEco\Zed\Payone\Persistence\PayoneQueryContainerInterface;
+
+abstract class AbstractPayoneRequestSender
```

Comment:
* Check the naming of a classes who extend this class, the suffix for them should be also `RequestSender`.

Reply:
Sure. thanks